include/swift/Serialization/ModuleFormat.h

```diff
@@ -52,7 +52,13 @@ const uint16_t SWIFTMODULE_VERSION_MAJOR = 0;
 /// describe what change you made. The content of this comment isn't important;
 /// it just ensures a conflict if two people change the module format.
 /// Don't worry about adhering to the 80-column limit for this line.
-const uint16_t SWIFTMODULE_VERSION_MINOR = 509; // [dynamic_lifetime] sil flag
+const uint16_t SWIFTMODULE_VERSION_MINOR = 510; // better string hash seed
```

Comment:
What's up with the end-of-line comment both on the LHS and the RHS of the patch? Seems like neither one belongs on this line?

Reply:
Ah.. it's the last change that causes us to bump the version?

Reply:
Yeah, see the comment above.